golang-job-type/go_wrapper/go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/TheRacetrack/plugin-go-job-type/golang-job-type
 
-go 1.16
+go 1.20
 
 require (
-	github.com/gorilla/mux v1.8.0
+    github.com/gin-gonic/gin v1.9.0
+    github.com/prometheus/client_golang v1.14.0
```

Comment:
Nitpick: I think you're mixing tabs with spaces in this file. Please unify this.

By the way, I see the indentations are odd in other files. Go has a nice built-in tool to fix this. Run 
`go fmt ./...` in a golang-job-type/go_wrapper directory.